### PR TITLE
Fix CI jobs with package update

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: ci-operator/jobs/openshift/ironic-agent-image/openshift-ironic-agent-image-main-presubmits.yaml [hunk 1]
```diff
@@ -478,6 +479,159 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )okd-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (ardaguclu) [inline ci-operator/jobs/openshift/ironic-agent-image/openshift-ironic-agent-image-main-presubmits.yaml:483]: Should this be true or false?

**Review by ardaguclu — COMMENTED** [review]
I have only one comment. Other than this, lgtm

File: ci-operator/jobs/openshift/ironic-agent-image/openshift-ironic-agent-image-main-presubmits.yaml (hunk 1 above)
Comment (elfosardo) [reply ci-operator/jobs/openshift/ironic-agent-image/openshift-ironic-agent-image-main-presubmits.yaml:483]: I'll fix it

**Pull request merged**